website/site/content/guide/user/_index.md

```diff
@@ -9,49 +9,103 @@ This guide is an overview of the basic features to get you started. Focalboard i
 
 [Let us know](/feedback/) if there's a feature you are looking for, or [connect with our vibrant community](/contribute/getting-started/) to help bring it to life.
 
+## Enabling Focalboard
```

Comment:
Thanks Justine! I've added this section to a [new page under download \ mattermost here](https://github.com/mattermost/focalboard/blob/main/website/site/content/download/mattermost/_index.md). You can remove it from this users guide page.

Reply:
Great! I've removed this section in this branch - let me know if that's correct? Then when we merge this the new page you created will be applied and I'll cross-check that it's correct. 

We just need to give Amy the right URL for the release notes - and that will be the new page you created.